### ISSUE TITLE
Accessibility-> on Home page, link to "sitesandtrailsbc.ca" is not identified and read as a link by screen reader

### ISSUE THREAD

**Issue by manuji** [bcgov/nr-rec-resources#656]
**Describe the bug**
On Home page of the beta site, under the section "Welcome to the new Recreation Sites and Trails BC beta site!", the url to the legacy site "sitesandtrailsbc.ca" is not recognized as a link by the screen reader and get read in a bizarre way that is hard to understand by the user using the screen reader.  
This issue fails the [4.1.2 Name, Role, Value (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html) criterion.

**To Reproduce**
1. Go to 'https://d1vrz2gt4nmfe5.cloudfront.net/'
2. Using a screen reader navigate through the Home page
3. Check how the link to the legacy site found under the section "Welcome to the new Recreation Sites and Trails BC beta site!" is read by the screen reader. 

**Expected behavior**
The link must have proper ARIA labels and tags so that screen reader can recognize links and read them to the user in a user friendly manner. 

**Screenshots**

<img width="1390" alt="Image" src="https://github.com/user-attachments/assets/f760f00f-c44b-418e-922b-2651fb0b4b70" />

**Additional context**
Tested on MacOS on Chrome using VoiceOver.
